pkg/controller/worker/firewall_reconcile.go

```diff
@@ -126,6 +146,18 @@ func (a *actuator) ensureFirewallDeployment(ctx context.Context, log logr.Logger
 		deploy.Spec.Template.Spec.LogAcceptedConnections = d.infrastructureConfig.Firewall.LogAcceptedConnections
 		deploy.Spec.Template.Spec.SSHPublicKeys = []string{sshKey}
 
+		if networkAccessType == apismetal.NetworkAccessForbidden {
```

Comment:
There is now a validation that enforces a network access type in the firewall-controller-manager. Please default to baseline.

Reply:
Moved the line out of the if statement